apps/model/model.js

```diff
@@ -508,43 +512,56 @@ function runPredict(key) {
         fullResCvs.width = l_img.width;
         fullResCvs.getContext('2d').drawImage(l_img, 0, 0);
 
-        let imgData = fullResCvs.getContext('2d').getImageData(0,0,fullResCvs.width,fullResCvs.height);
 
-        const img = tf.browser.fromPixels(imgData).toFloat();
+        let imgData = fullResCvs.getContext('2d').getImageData(0,0,fullResCvs.width,fullResCvs.height);
         let img2;
+       tf.tidy(()=>{
+        const img = tf.browser.fromPixels(imgData).toFloat();
+        
         if (input_channels == 1) {
           img2 = tf.image.resizeBilinear(img, [image_size, image_size]).mean(2);
         } else {
           img2 = tf.image.resizeBilinear(img, [image_size, image_size]);
         }
+        return img2;
+          });
         let scaleMethod = $UI.filter? $UI.filter.status: 'norm';
         console.log(scaleMethod);
-
+         
         let normalized;
         if (scaleMethod == 'norm') {
           // Pixel Normalization: scale pixel values to the range 0-1.
 
           let scale = tf.scalar(255);
           normalized = img2.div(scale);
+          scale.dispose();
 
         } else if (scaleMethod == 'center') {
           // Pixel Centering: scale pixel values to have a zero mean.
-
+          
           let mean = img2.mean();
           normalized = img2.sub(mean);
+          mean.dispose();
           // normalized.mean().print(true); // Uncomment to check mean value.
           // let min = img2.min();
           // let max = img2.max();
           // let normalized = img2.sub(min).div(max.sub(min));
         } else {
           // Pixel Standardization: scale pixel values to have a zero mean and unit variance.
-         
+            tf.tidy(()=> {
```

Comment:
Where is `normalized` returned to

Reply:
to keep ``` tf.tidy() ``` from destroying the tensor it needs to be returned. ``` normalized ``` is being used afterwards so it doesn't need to be destroyed.

Reply:
The returned object has to be assigned

Reply:
I am using ``` tf.keep() ``` instead of return. Does this solve the issue?

Reply:
It still needs to be assigned, please refer the usage here: https://js.tensorflow.org/api/latest/#keep